backend/src/main/java/org/cryptomator/hub/filters/VaultAdminOnlyFilterProvider.java

```diff
@@ -66,12 +68,17 @@ JWTVerifier buildVerifier(Algorithm algorithm) {
 		return verification(algorithm).build();
 	}
 
+	private Verification verification(Algorithm algorithm) {
+		return verification(algorithm, Instant.now());
+	}
+
 	//visible for testing
-	Verification verification(Algorithm algorithm) {
+	Verification verification(Algorithm algorithm, Instant now) {
 		return JWT.require(algorithm) //
-				.withClaim(RegisteredClaims.ISSUED_AT, (claim, jwt) -> jwt.getIssuedAt() != null) //
-				.withClaim(RegisteredClaims.NOT_BEFORE, (claim, jwt) -> jwt.getNotBefore() != null) //
-				.withClaim(RegisteredClaims.EXPIRES_AT, (claim, jwt) -> jwt.getExpiresAt() != null);
+				.withClaim(RegisteredClaims.ISSUED_AT, (claim, jwt) -> jwt.getIssuedAt() != null //
+						&& !jwt.getIssuedAt().before(Date.from(now.minus(REQUEST_LEEWAY_IN_SECONDS, ChronoUnit.SECONDS))) //
+						&& !jwt.getIssuedAt().after(Date.from(now.plus(REQUEST_LEEWAY_IN_SECONDS, ChronoUnit.SECONDS)))) //
```

Comment:
careful: these are `java.sql.Date`

Reply:
Fixed in https://github.com/cryptomator/hub/pull/164/commits/42e69c0ddc2006e41a41080a28bfd94e7029d58c